src/res_printer.ml

```diff
@@ -2815,7 +2819,7 @@ and printExpression (e : Parsetree.expression) cmtTbl =
             Doc.softLine;
             spread;
             Doc.join ~sep:(Doc.concat [Doc.text ","; Doc.line])
-              (List.map (fun row -> printRecordRow row cmtTbl) rows)
+              (List.map (fun row -> printRecordRow row cmtTbl punningAllowed) rows)
```

Comment:
Instead of passing `punningAllowed` to every row, would it be "clearer" if we special cased the one-element record?

```js
// pseudo-code

if (one-element-record) {
  …
} else {
  List.map printRecordRow…
}
```

What do you think about this?

Reply:
Just tried that here: https://github.com/cknitt/syntax/commit/d83c61f31505c07e3ca737d519e9292c2c5c834d

Do you think it is an improvement?